build.gradle

```diff
@@ -113,6 +113,7 @@ publishing {
             pom {
                 name = pluginName
                 description = pluginDescription
+                groupId = "org.opensearch.plugin"
```

Comment:
Do we need double quote here?

Reply:
Yes @heemin32 its required, else if would fail to infer and look for a value of org.opensearch.plugin (var), which is in not assigned.